.gitignore

```diff
@@ -16,3 +18,4 @@ config.json
 poetry.lock
 .DS_Store
 *.log
+brija.py
```

Comment:
nitpick: no need to add random garbage to .gitignore, you can just remove the files from your directory (or add to your personal ~/.gitignore)